### DISCUSSION TITLE
[Feature] Album Add "Person/Places"

### DISCUSSION THREAD

**Discussion by superdong69** [immich-app/immich#11225] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I think having the ability to create an album, and add images by PEOPLE/Places would be an extremely useful feature. It should be able to auto populate when new photos of the people designated to the album get added.

Very useful for categorizing vacations, or content for ( in my case what I'm looking to do ) is to use with immich frame to only show people I want/events/ etc and not EVERY picture in my library.

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by droopified** [comment]
i had opened a tab to write the exact same thing but went to lunch and u already posted it lol. but yes similar to Plex's "Smart" playlist where if i make an album containing x face, always include all images with x face and auto populate. and can work with a combination of keywords like x person + beach. or something similar.

**Reply by superdong69** [reply]
Yes exactly! Great minds think alike!

**Comment by bo0tzz** [comment]
#1673
